### ISSUE TITLE
Add "show guide" button

### ISSUE THREAD

**Issue by EliWei** [layersoflondon/application#628]
Add an extra button for "Show guide" to the left. And change button from Help to Help Centre.

Please let Elisabeth know time estimate on this one before starting!

**Comment by EliWei** [comment]
Up to 0.5 day

**Comment by EliWei** [comment]
Please go ahead with this one

**Comment by edtjones** [comment]
Just need to style the link in the tools component. Assigning to @andyError

**Comment by edtjones** [comment]
There's a separate branch called feature/reopen-intro for this @andyError

**Comment by edtjones** [comment]
This is now merged into develpment @andyError - there's a link in the tools component which just need styles

**Comment by andyError** [comment]
Done in dev.
